watson-developer-cloud/discovery-components
# fix: offest issue in CIDocument

#### What do these changes do/fix?



#### How do you test/verify these changes?
This changes fixes the highlighting of the attributes in CIDocument, you can see it's working properly by running storybook `yarn storybook` and navigating to `CIDocument -> default`. select the highlights document and select defined terms under attributes you should see that  everything is highlighted correctly.

#### Have you documented your changes (if necessary)?
Yes
#### Are there any breaking changes included in this pull request?
No
